src/components/OrderedList/OrderedList.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Could you add comments to the top of this file? Like --

```suggestion
/**
 * @file Ordered list stories.
 * @copyright IBM Security 2019
 */
 
 import React from 'react';
```